### DISCUSSION TITLE
Is there a way to disable hmr (hot module reloading)?

### DISCUSSION THREAD

**Discussion by cdeutsch** [sveltejs/kit#8355] [Q&A]
I can't seem to find an answer on Google or by searching this repos docs or issues

**Comment by cdeutsch** [comment]
I think I found it?

```js
const config = {
  plugins: [sveltekit()],
    server: {
      hmr: false
    }
};
```

![image](https://user-images.githubusercontent.com/404765/210888469-b86eee97-4ed9-4c23-8930-f52f6141531d.png)

**Reply by samal-rasmussen** [reply]
Thanks. Unfortunately there is no page reload when hmr is false. Having to manually refresh the page is a drag.

**Reply by samal-rasmussen** [reply]
Ok seems like you need to add a custom plugin in order to get reload on save: https://stackoverflow.com/questions/71008900/vite-js-how-do-i-enable-full-page-reload-on-ctrls

```ts
import { sveltekit } from '@sveltejs/kit/vite';
import { defineConfig, type PluginOption } from 'vite';

const full_reload_always: PluginOption = {
	name: 'full-reload-always',
	handleHotUpdate({ server }) {
		server.ws.send({ type: 'full-reload' });
		return [];
	},
};

export default defineConfig({
	plugins: [sveltekit(), full_reload_always],
	test: {
		include: ['src/**/*.{test,spec}.{js,ts}'],
	},
});
```

Notice that hmr is not disabled now, because disabling hmr will actually also disable the custom full reload always plugin...
